src/saltext/vmware/modules/tag.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License: Apache-2.0
+import logging
+
+import salt.exceptions
+import saltext.vmware.utils.common as utils_common
+import saltext.vmware.utils.connect as connect
+
+log = logging.getLogger(__name__)
+
+try:
+    from pyVmomi import vim
+
+    HAS_PYVMOMI = True
+except ImportError:
+    HAS_PYVMOMI = False
+
+
+__virtualname__ = "vmware_tag"
+__func_alias__ = {"list_": "list"}
+
+
+def __virtual__():
+    if not HAS_PYVMOMI:
+        return False, "Unable to import pyVmomi module."
+    return __virtualname__
+
+
+def create(tag_name, category_id, description=""):
+    """
+    Create a new tag.
+
+    tag_name
+        Name of tag.
+
+    category_id
+        (string) Category ID of type: com.vmware.cis.tagging.Tag.
```

Comment:
It would be helpful to provide actual examples of what this even means. I think this is a copy from the API docs but... we can definitely do better here.